pkg/totalaccountwatcher/totalaccountwatcher.go

```diff
@@ -26,6 +26,9 @@ var ErrAwsAccountLimitExceeded = errors.New("AccountLimitExceeded")
 // TotalAccountWatcher global var for TotalAccountWatcher
 var TotalAccountWatcher *totalAccountWatcher
 
+// TotalAccountWatcherInitialized ensures async initialization triggers only once
+var TotalAccountWatcherInitialized bool = false
```

Comment:
How could it initialize more than once?

Reply:
I had a pair programming session yesterday with @rogbas and he mentioned 

```go
go totalaccountwatcher.TotalAccountWatcher.Start(log, stopCh, awsClient, totalWatcherInterval)
```

Starts the method in an async way,  so we need make sure we do not constantly initialising the TAW. So the first time we start the go routine, initialize is running, all fine. But after a period of time (e.g. 15 m), we run it for a second time, we don't need to re-initialize the watcher.

I am puzzled now though, because @2uasimojo observations seems correct. How this can run a second time?

Reply:
Right; this isn't running in a controller, it's running as part of startup, so it should only be hit once.

Reply:
Yes, nvm, the Start function will be called only once, there's no need for having this check to prevent running Initialize twice.